python/tk_multi_workfiles/file_save_form.py

```diff
@@ -68,13 +68,20 @@ def __init__(self, parent=None):
             preview_colour = font_colour.darker(140)
         self._preview_colour = (preview_colour.red(), preview_colour.green(), preview_colour.blue())
 
+        self._initializing = False
         try:
             # doing this inside a try-except to ensure any exceptions raised don't 
             # break the UI and crash the dcc horribly!
+            self._initializing = True
```

Comment:
We can always move the first assignment outside the if, but the second example you provided isn't equivalent. If something goes wrong during start_preview_update in your example, the exception will bubble out of the __init__.